x-pack/plugins/triggers_actions_ui/public/application/constants/index.ts

```diff
@@ -88,3 +88,23 @@ export const LOCKED_COLUMNS = [
 export const RULE_EXECUTION_DEFAULT_INITIAL_VISIBLE_COLUMNS = [...LOCKED_COLUMNS.slice(1)];
 export const GLOBAL_EXECUTION_DEFAULT_INITIAL_VISIBLE_COLUMNS = ['rule_name', ...LOCKED_COLUMNS];
 export const DEFAULT_NUMBER_FORMAT = 'format:number:defaultPattern';
+
+export const CONNECTOR_EXECUTION_LOG_COLUMN_IDS = [
+  'connector_id',
+  'space_ids',
+  'id',
+  'timestamp',
+  'execution_duration',
+  'status',
+  'connector_name',
+  'message',
+  'execution_duration',
+  'schedule_delay',
+  'timed_out',
```

Comment:
nit: execution_duration is here twice and I don't see a column for connector_id?

Reply:
Resolved in this commit e3ed9964f50e9470aba39f70beb5f8fb4deed863